hyperlink/test/common.py

```diff
@@ -0,0 +1,70 @@
+
+
+from unittest import TestCase
+
+
+class HyperlinkTestCase(TestCase):
```

Comment:
Actually, this is really only a Python 2.6 compatibility concern. Do we need to maintain Python 2.6 compat?